picard/ui/scripteditor.py

```diff
@@ -259,16 +246,22 @@ class ScriptEditorPage(PicardDialog):
             "file_naming_format",
             DEFAULT_FILE_NAMING_FORMAT,
         ),
+        ListOption(
+            "setting",
+            "file_naming_scripts",
+            [],
+        ),
     ]
 
+    FILE_ERROR_IMPORT = N_('Error importing "%s". %s.')
+    FILE_ERROR_DECODE = N_('Error decoding "%s". %s.')
+    FILE_ERROR_EXPORT = N_('Error exporting file "%s". %s.')
```

Comment:
I think those should be moved near where they are used, to provide better context to translators.
I don't think they should be class-level attributes.